mimic.py

```diff
@@ -0,0 +1,69 @@
+from sys import stdout
+
+
+class Node:
+    def __init__(self, value, prev=None, next=None):
+        self.value = value
+        self.prev = prev
+        self.next = next
+
+
+class MyList:
+    def __init__(self):
+        self.start = Node('start', None, None)
+        self.end = Node('end', self.start, None)
+        self.start.next = self.end
+        self.list_len = 0
+
+    def append(self, val):
+        new_node = Node(val, self.end.prev, self.end)
+        self.end.prev.next = new_node
+        self.end.prev = new_node
+        self.list_len += 1
+
+    def print(self):
+        stdout.write('[')
+        node = self.start.next
+        while node != self.end:
+            stdout.write('{},'.format(node.value))
+            node = node.next
+        stdout.write('\b]\n')

```

Comment:
What's `\b`?

Reply:
Backspace. But I converted all to print with f-strings. Done